script.js

```diff
@@ -251,3 +251,51 @@ function execute() {
 /* event triggers */
 container.addEventListener('click', execute);
 PopWindow.addEventListener('click', close);
+
+// Form Validation
+const form = document.getElementById('form');
+const error = document.querySelector('.error');
+const name = document.getElementById('name');
+const email = document.getElementById('email');
+const message = document.getElementById('message');
+
+form.addEventListener('submit', (e) => {
+  const message = [];
+  if (email.value !== email.value.toLowerCase()) {
+    e.preventDefault();
+    message.push('E-mail has to be in lowercase letters !!!');
+  }
+  if (message.length > 0) {
+    error.innerText = message.join(', ');
+  }
+});
+
+let formLocal = { name: '', email: '', message: '' };
+if (localStorage.formLocal) {
+  formLocal = JSON.parse(localStorage.formLocal);
+  name.value = formLocal.name;
+  email.value = formLocal.email;
+  message.value = formLocal.message;
+}
+form.addEventListener('input', () => {
+  localStorage.formLocal = JSON.stringify(formLocal);
+  formLocal.name = name.value;
+  formLocal.email = email.value;
+  formLocal.message = message.value;
+});
+
+// Form Validation
+const formContainer = document.getElementsByClassName('form-container');
+const check = document.querySelector('.notification');
+const mail = document.getElementsByClassName('email');
+
+formContainer.addEventListener('submit', (e) => {
+  const message = [];
+  if (mail.value !== mail.value.toLowerCase()) {
+    e.preventDefault();
+    message.push('E-mail has to be in lowercase letters !!!');
+  }
+  if (message.length > 0) {
+    check.innerText = message.join(', ');
+  }
+});
```

Comment:
Hello @team, for some reason this condition is not working as it should. I tested it with an email in uppercase and it got submitted which shouldn't be the case as seen below 👇🏻. 

- [ ] Please look at it again and find what the bug is! When the user submits the form, you check if the email is in lower case. If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.


https://user-images.githubusercontent.com/4247740/152636378-8390a09f-ffbb-41d8-be1a-e58819d5b536.mp4